operators/hpe-csi-operator/sources/hpe-csi-operator.csv.yaml

```diff
@@ -0,0 +1,198 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    features.operators.openshift.io/csi: "true"
+    features.operators.openshift.io/disconnected: "false"
+    features.operators.openshift.io/fips-compliant: "false"
+    features.operators.openshift.io/proxy-aware: "false"
+    features.operators.openshift.io/tls-profiles: "false"
+    features.operators.openshift.io/token-auth-aws: "false"
+    features.operators.openshift.io/token-auth-azure: "false"
+    features.operators.openshift.io/token-auth-gcp: "false"
+    features.operators.openshift.io/cnf: "false"
+    features.operators.openshift.io/cni: "false"
+    operatorhub.io/ui-metadata-max-k8s-version: "1.29"
+    olm.skipRange: ">=1.0.0 <%SEMVER%"
+    alm-examples: '[]'
+    capabilities: Basic Install
+    categories: Storage
+    certified: "true"
+    description: >-
+      A Container Storage Interface (CSI) driver for HPE storage platforms. The CSI driver
+      allows you to use HPE storage with your preferred container orchestrator.
+    support: "HPE"
+    repository: "https://github.com/hpe-storage/co-deployments"
+  name: hpe-csi-operator.v0.0.0
+  namespace: placeholder
+spec:
+  apiservicedefinitions: {}
+  customresourcedefinitions: {}
+  description: Bar
+  displayName: Foo
+  icon:
+  - base64data: ""
+    mediatype: ""
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  maturity: stable
+  provider:
+    name: HPE
+  version: 0.0.0
+  displayName: HPE CSI Operator for Kubernetes
+  description: >
+    The HPE CSI Operator for Kubernetes deploys the HPE CSI Driver on
+    Kubernetes and OpenShift for dynamic provisioning of persistent
+    volumes on HPE storage systems.
+
+    The HPE CSI Driver for Kubernetes leverages HPE storage platforms
+    to provide scalable and persistent storage for stateful applications.
+
+    Please refer to our CSI driver [documentation](https://scod.hpedev.io/csi_driver/index.html)
+    for supported HPE platforms and full list of supported CSI features.
+
+    ## Installation
+
+    Refer to the HPE CSI Operator for Kubernetes [documentation](https://scod.hpedev.io/csi_driver/deployment.html#operator).
+
+  maturity: stable
+  version: %SEMVER%
+  minKubeVersion: 1.26.0
```

Comment:
This version changes too. It would be better to have regular expression here as well.

Reply:
minKubeVersion needs to be considered for each release. I changed to 1.25.0 to support OpenShift 4.12.